src/components/modal/v2/parts/views/ShortTerm/Content.jsx

```diff
@@ -0,0 +1,72 @@
+        qualifying = useServerData().views[0].meta.qualifying;
+        periodicPayment = useServerData().views[0].meta.periodicPayment;
+    }
+
+    const [expandedState] = useState(false);
+    return (
+        <Fragment>
+            <style>{styles._getCss()}</style>
+            <div className="content__container">
+                <main className="main">
+                    <div className="content__body" ref={contentBodyRef}>
+                        <div className="dynamic__container">
+                            <div className="content__row dynamic">
+                                <div className="content__col">
+                                    <div className="content__row donuts">
+                                        <div
+                                            className={`donuts__container ${
+                                                qualifying === 'true' ? 'donuts__qualifying' : 'donuts__non_qualifying'
+                                            }`}
+                                        >
+                                            {donutTimestamps.map((val, index) => (
+                                                <Donut
+                                                    qualifying={qualifying}
+                                                    periodicPayment={periodicPayment}
+                                                    currentNum={index + 1}
+                                                    timeStamp={donutTimestamps[index]}
+                                                >
+                                                    {qualifying &&
+                                                        `${((1 / donutTimestamps.length) * 100).toFixed(0)}%`}
+                                                </Donut>
+                                            ))}
+                                        </div>
+                                    </div>
+                                    <Instructions instructions={instructions} expandedState={expandedState} />
+                                </div>
+                                <div className={`content__col ${expandedState ? '' : 'collapsed'}`}>
+                                    <div className="branded-image">
+                                        {/* TODO: update from temp desktop image */}
+                                        <Icon name="pi4-image" />
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <div className={`content__row disclosure ${expandedState ? '' : 'collapsed'}`}>
+                            <p>{disclosure}</p>
+                        </div>
+                        <div className="content__row productLink">
+                            <div className="productLink__container">
+                                <ProductListLink>{linkToProductList}</ProductListLink>
```

Comment:
```suggestion
                                <ProductListLink>{renderProductListLink()}</ProductListLink>
```
From the above comment about adding the ```renderProductListLink``` function.